composer.json

```diff
@@ -48,7 +48,7 @@
         }
     },
     "require": {
-        "php": "^7.4.1",
+        "php": "^8.1",
```

Comment:
Many developers will want to burn you alive for this change. It is really pitty there is no middle step with support of both PHP versions (`^7.4.1|^8.1`)

Reply:
Hi @s3tezsky :blush: Yeah, it would be nice to have the middle step, but you know, PHP 7.4 will end the support soon one way or another. Moreover, we would have to come up with a workaround for Roave/Better-reflection which has no middle step as well so in our codebase, we would have to require both versions and check the existence of particular classes, etc. I would say that does not worth it, the next release will be definitely a major, basically just with the upgraded PHP and related dependencies so I believe nobody will want to burn us alive :slightly_smiling_face:

Reply:
Burning alive was just a hyperbolic meaning 😁. Just remember how even smaller upgrades can get you mad sometimes... With this change you switch your PHP version and pray for better tomorows and painless upgrade.
I just wanted to write it is a pitty. You guys know what you are doing 😉.
So the SSFW X is comming soon?

Reply:
> Burning alive was just a hyperbolic meaning

I know I know :slightly_smiling_face: 

> So the SSFW X is comming soon?

Oh, what a cool name for the next version :grimacing: we should definitely use it somehow :smile: And yes, we have done almost all prerequisites so we should start the release process soon :wink:

Reply:
> Sadly the middle step is not achievable easily.

If it were easy, everyone do it 😁.

> All in all too much work for a PHP version that doesn't have an active support and stops receiving security updates in a few months.

I was pointing mainly on developer experience during upgrading. Some extra time of hacking could save amount of time and gray hair of them.

PS: I respect your decision. Nice job with the upgrade 👍